op-challenger/game/fault/types/position.go

```diff
@@ -81,11 +122,23 @@ func (p Position) Defend() Position {
 }
 
 func (p Position) Print(maxDepth int) {
-	fmt.Printf("GIN: %4b\tTrace Position is %4b\tTrace Depth is: %d\tTrace Index is: %d\n", p.ToGIndex(), p.indexAtDepth, p.depth, p.TraceIndex(maxDepth))
+	fmt.Printf("GIN: %4b\tTrace Position is %4b\tTrace Depth is: %d\tTrace Index is: %d\n", p.ToGIndex(), p.indexAtDepth.Uint64(), p.depth, p.TraceIndex(maxDepth))
```

Comment:
```suggestion
	fmt.Printf("GIN: %4b\tTrace Position is %4b\tTrace Depth is: %d\tTrace Index is: %d\n", p.ToGIndex(), p.indexAtDepth, p.depth, p.TraceIndex(maxDepth))
```

We should never be calling `.UInt64()` on the big ints or we're going to introduce bugs.